src/components/Heading/Heading.test.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { renderWithTheme } from '../../../jest/utils';
+import Heading from '.';
+import Section from '../Section';
+
+describe('Heading', () => {
+  it('renders as a h2 by default', () => {
+    const { getByText } = renderWithTheme(<Heading>Hello</Heading>);
+    expect(getByText('Hello').tagName).toEqual('H2');
+  });
+
+  it('allows rendering a h1', () => {
+    const { getByText } = renderWithTheme(<Heading as="h1">Hello</Heading>);
+    expect(getByText('Hello').tagName).toEqual('H1');
+  });
+
+  it('does not render above an h6', () => {
+    const { getByText } = renderWithTheme(
+      <div>
+        <Heading>h2</Heading>
+        <Section>
```

Comment:
I thought `section` represents a generic standalone section of a document and not that during nested sections we should increase the heading level!

Reply:
I think that was the whole point of this PR 😄  That you don't have to think about `h1` vs `h2`, but that Pounce would do that for you